### ISSUE TITLE
Add link to Vuejs TypeScript course 

### ISSUE THREAD

**Issue by alexjoverm** [vuejs/vuejs.org#1099]
Hi! I've seen somewhere else in the docs links to Egghead courses, for example to the [Webpack's one](https://egghead.io/courses/using-webpack-for-production-javascript-applications).

Would you like to add [Use TypeScript to Develop Vuejs Applications](https://egghead.io/courses/use-typescript-to-develop-vue-js-web-applications)? I can send a PR if wanted.

Thanks!

**Comment by LinusBorg** [comment]
I'm not aware that we would link to that course from our documentation/guide. Can you point me to that location?

Or maybe you are referring to [awesome-vue](https://github.com/vuejs/awesome-vue)? In that case, open a PR in that repository, see its CONTRIBUTING.md for guidelines.

**Comment by alexjoverm** [comment]
https://vuejs.org/v2/guide/single-file-components.html#For-Users-New-to-Module-Build-Systems-in-JavaScript

**Comment by LinusBorg** [comment]
Ha, honestly didn't know that.

Since the docs have a seperate page for tpescript, I think we would be open to  a PR adding it there?

/cc @chrisvfritz

**Comment by chrisvfritz** [comment]
@alexjoverm The general rule I've been following is to avoid linking to paid resources (even made by ourselves), because it introduces a potential conflict of interests. We made an exception with that Webpack course, because at the time of publishing, the Webpack docs were still in very poor shape and free, high-quality resources were scarce. 

Since then, the docs have gotten much better and Sean Larkin has also published Webpack Academy, so I've just removed the Egghead link and updated our linked resources. 🙂  That said, we do really appreciate your contribution to helping people learn to use Vue + TypeScript and a PR would definitely be welcome on Vue Awesome!
